script/dash-lint.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+CURRENT_FILE=$(basename "${BASH_SOURCE}")
+ROOT_DIR=$(git rev-parse --show-toplevel)
+BAD_NAMES=$(grep -ril "bucketblocker" "$ROOT_DIR" --exclude "$CURRENT_FILE")
+
+if [ -n "$BAD_NAMES" ]; then
+    printf "The following files contain the incorrect string 'bucketblocker':\n\n"
+    echo "$BAD_NAMES"
+    printf "\nbucket-blocker has a dash in it, and inconsistencies in the codebase can cause major problems, please correct the spelling.\n"
```

Comment:
Can the two printf statements be merged into one, and then the output redirected to stderr? Something like

```bash
 printf "Blablabla [...], use 'bucket-blocker' instead:\n\n" >&2
 echo "$BAD_NAMES"
```
It's conventional in UNIX-like command tools so that the output of a program can be piped into another command.